tools/sidekick/config.json

```diff
@@ -1,4 +1,6 @@
 {
+  "project": "HOMEPAGE",
```

Comment:
I see some are all capital letters (BACOM, DC), some are not (Milo, Creative Cloud). Not sure if it makes any difference, just wanted to call it out.

Reply:
thanks for checking.
case does not matter.